content/pages/burials-and-memorials/application/530.md

```diff
@@ -3,24 +3,12 @@ title: Apply for burial benefits
 entryname: burials
 layout: page-react.html
 ---
-<div id="main">
-  <nav aria-label="Breadcrumb" aria-live="polite" class="va-nav-breadcrumbs"
-  id="va-breadcrumbs">
-    <ul class="row va-nav-breadcrumbs-list columns" id="va-breadcrumbs-list">
-      <li><a href="/">Home</a></li>
-      <li><a href="/burials-and-memorials/"> Burials and Memorials </a></li>
-      <li><a href="/burials-and-memorials/survivor-and-dependent-benefits/"> Survivors and Dependents </a></li>
-      <li><a aria-current="page" href="/burials-and-memorials/survivor-and-dependent-benefits/burial-costs/">Handling Burial Costs (Allowances)</a></li>
-    </ul>
-  </nav>
-  <div class="section">
-    <div id="react-root">
-      <div class="loading-message">
-        <div class="loading-indicator-container">
-          <div class="loading-indicator" role="progressbar" aria-valuetext="Please wait while we load the application for you." tabIndex="0"></div> Please wait while we load the application for you.
-        </div>
-      </div>
-    </div>
-  </div>
-</div>
-
+<nav aria-label="Breadcrumb" aria-live="polite" class="va-nav-breadcrumbs"
+id="va-breadcrumbs">
+  <ul class="row va-nav-breadcrumbs-list columns" id="va-breadcrumbs-list">
+    <li><a href="/">Home</a></li>
+    <li><a href="/burials-and-memorials/"> Burials and Memorials </a></li>
+    <li><a href="/burials-and-memorials/survivor-and-dependent-benefits/"> Survivors and Dependents </a></li>
+    <li><a aria-current="page" href="/burials-and-memorials/survivor-and-dependent-benefits/burial-costs/">Handling Burial Costs (Allowances)</a></li>
+  </ul>
+</nav>
```

Comment:
@1Copenut I wonder if there's any chance of us being able to move the breadcrumbs into the React layout. We have a breadcrumbs template in the `includes /` directory that would be nice if we could leverage.